lib/sourceParams.js

```diff
@@ -131,6 +131,11 @@ module.exports = function (app) {
   } else if (flags.attachValidator) {
     params.htmlValidator.separateProcess.enable = false
   }
+  if (process.env.IS_DETACHED === 'true') {
```

Comment:
@JoshL18 we talked about this offline but I like the idea of namespacing ENV to the project, like `ROOSEVELT_IS_DETACHED`. Others are welcome to comment on this